specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/virtualMachineScaleSet.json

```diff
@@ -3948,6 +3948,61 @@
       },
       "description": "The configuration parameters used while performing resilient VM deletion."
     },
+    "ScheduledEventsPolicy": {
+      "type": "object",
+      "description": "Specifies Redeploy,Reboot and scheduledEventsAdditionalPublishingTargets Scheduled Event related configurations.",
+      "properties": {
+        "UserInitiatedRedeploy": {
```

Comment:
Camel case for all of these please

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/definitions-properties-names-camel-case.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/definitions-properties-names-camel-case.md)

Reply:
done